frontend-react/src/config/endpoints/waters.ts

```diff
@@ -54,10 +54,11 @@ export enum WatersUrls {
 The error codes map to the error types specified in the serializer
  */
 export enum ErrorCodeTranslation {
-    INVALID_HL7_MESSAGE_VALIDATION = "Invalid entry for field.",
-    INVALID_HL7_MESSAGE_DATE_VALIDATION = "Invalid entry for field. Reformat to either the HL7 v2.4 TS or ISO 8601 standard format.",
-    INVALID_HL7_MESSAGE_FORMAT = "",
-    INVALID_HL7_PHONE_NUMBER = "The string supplied is not a valid phone number. Reformat to a 10-digit phone number (e.g. (555) 555-5555).",
+    INVALID_HL7_MSG_VALIDATION = "Invalid entry for field.",
+    INVALID_MSG_PARSE_DATE = "Invalid entry for field. Reformat to either the HL7 v2.4 TS or ISO 8601 standard format.",
+    INVALID_HL7_MSG_FORMAT_INVALID = "",
+    INVALID_MSG_PARSE_TELEPHONE = "The string supplied is not a valid phone number. Reformat to a 10-digit phone number (e.g. (555) 555-5555).",
```

Comment:
Has it been verified that this is explicitly the only phone number format that is supported?

Reply:
I don't *think* it is. We use a google library for processing phone numbers and that library supports different formats. This is a front-end message so I just left it as is. I think it's fine because the format the message is suggesting will fix the issue the client is having.